kops/kops.go

```diff
@@ -1,57 +1,30 @@
 package kops
 
 import (
+	"bytes"
 	"encoding/json"
-	"errors"
 	"fmt"
-	"io/ioutil"
-	"os"
-	"strings"
-
 	clusteroperatorv1alpha1 "github.com/infobloxopen/cluster-operator/pkg/apis/clusteroperator/v1alpha1"
 	"github.com/infobloxopen/cluster-operator/utils"
+	"github.com/spf13/viper"
 	"gopkg.in/yaml.v2"
+	"io/ioutil"
+	"strings"
 )
 
 type KopsCmd struct {
```

Comment:
Merging from master will add a new 'path' element of the struct which allows for the binary to run from the defined path. That may be a variable you want to include in the Viper setup